source/pattern-library/data-visualization/heat-map/index.md

```diff
@@ -1,60 +1,7 @@
 ---
-title: Heat Map
-author: dlabrecq
-layout: page-tabs
+author: lhinson
+layout: page-pattern
+overview: patterns/data-visualization/heat-map/overview.md
+design: patterns/data-visualization/heat-map/design.md
+code: /components/angular-patternfly/dist/docs/partials/api/patternfly.charts.directive.pfHeatMap.html
```

Comment:
I'll change the frontmatter to `code_angular` to allow for the addition of `code_core` later.